.github/DISCUSSION_TEMPLATE/new-type.yml

```
@@ -0,0 +1,50 @@
+title: "Add: "
+labels: ["New type"]
+body:
+- type: input
+  id: name
+  attributes:
+    label: Type name
+    description: The name of the terms type you would like to add.
+    placeholder: Business Privacy Policy
+  validations:
+    required: true
+- type: textarea
+  id: alternative-type-names
+  attributes:
+    label: Alternative type names
+    description: Different names for the same terms type.
+    placeholder: Business Users Privacy Policy, Commercial Privacy Policy
+- type: input
+  id: writer
+  attributes:
+    label: Writer
+    description: The writer of the document, in most cases the service provider itself.
+    placeholder: intermediation service provider
```

As you said: “in most cases the service provider itself”,  and this part is always very confusing for newcomers, so let's keep it simple with a sensible default 🙂 
```suggestion
    placeholder: service provider
```